content/ch-quantum-hardware/measuring-quantum-volume.ipynb

```diff
@@ -1116,7 +1116,7 @@
     "The *achievable depth* $d(m)$ is the largest $d$ such that we are confident that $h_d > 2/3$. In other words,\n",
     "\n",
     "\n",
-    "$$ h_1,h_2,\\dots,h_{d(m)}>2/3 \\text{ and } h_{d(m+1)} \\leq 2/3$$\n",
+    "$$ h_1,h_2,\\dots,h_{d(m)}>2/3 \\text{ and } h_{d(m)+1} \\leq 2/3$$\n",
```

Comment:
I don't think the original subscript is wrong...

Reply:
Under **Step 6: Calculate the achievable depth** (https://qiskit.org/textbook/ch-quantum-hardware/measuring-quantum-volume.html#Step-6:-Calculate-the-achievable-depth) 
The textbook mentions h_{d(m+1)} as before but like the issue this references mentions, on page 2 of the reference paper the paper mentions h_{d(m)+1} (https://arxiv.org/pdf/1811.12926.pdf) linked under **References** (https://qiskit.org/textbook/ch-quantum-hardware/measuring-quantum-volume.html#References)

Reply:
Ok, sorry, I was mistaken... as Quantum Volume (QV) circuits satisfy `d(m)=m` so `d(m+1)=d(m)+1`, and  are square shaped.  But, since they did not actually define the QV until the next paragraph, your fix is indeed correct.